src/main/java/gameframework/motion/MoveStrategyRandom.java

```diff
@@ -13,7 +13,7 @@
 
 	@Override
 	public SpeedVector getSpeedVector() {
-		int i = random.nextInt(5);
```

Comment:
are you fixing a bug here? What could have been the impact of this bug? If you feel so inclined, please add the unit tests that make sure the bug don't reappear.